lib/exceptions.h

```diff
@@ -28,9 +28,9 @@ limitations under the License.
 namespace Util {
 
 // colors to pretty print messages
-constexpr char ANSI_RED[] = "\e[31m";
-constexpr char ANSI_BLUE[] = "\e[34m";
-constexpr char ANSI_CLR[] = "\e[0m";
+constexpr char ANSI_RED[] = "\\e[31m";
```

Comment:
`\\e` is incorrect and it should be `\33` .

Reply:
Right, sorry about that, I meant to test that and then forgot. Should be fixed now.